docs/docker.rst

```diff
@@ -0,0 +1,84 @@
+.. inclusion-marker-start-do-not-remove
+
+Horovod in Docker
+=================
+
+To streamline the installation process on GPU machines, we have published the reference `Dockerfile <../Dockerfile>`__ so
+
+you can get started with Horovod in minutes. The container includes `Examples <../examples>`__ in the `/examples`
+directory.
+
+Pre-built docker containers with Horovod are available on `DockerHub <https://hub.docker.com/r/horovod/horovod>`__.
+
+**Building**
+
+Before building, you can modify `Dockerfile` to your liking, e.g. select a different CUDA, TensorFlow or Python version.
+
+.. code-block:: bash
+
+    $ mkdir horovod-docker
+    $ wget -O horovod-docker/Dockerfile https://raw.githubusercontent.com/horovod/horovod/master/Dockerfile
+    $ docker build -t horovod:latest horovod-docker
+
+
+**Running on a single machine**
+
+After the container is built, run it using `nvidia-docker <https://github.com/NVIDIA/nvidia-docker>`__.
+
+**Note**: you can replace `horovod:latest` with the `specific <https://hub.docker.com/r/horovod/horovod/tags>`__ pre-build
+Docker container with Horovod instead of building it by yourself
```

Comment:
Add a period at the end of the sentence

Reply:
Done! Your eye for detail is amazing, thank you!